frontend/src/views/CreateProposal.vue

```diff
@@ -90,12 +90,24 @@ export default {
         return callback(new Error('Please input a number of incentives'))
       } else {
         if (Number(value) > 1000000) {
-          return callback(new Error('No more than 1000,000 BOS'))
+          return callback(new Error('No more than 1,000,000.0000 BOS'))
         } else {
           callback()
         }
       }
     }
+    const checkPropName = (rule, value, cb) => {
+      if (value === '') {
+        return cb(new Error('Please input proposal name'))
+      } else {
+        const regex = /^([a-z]|[1-5]){12}$/g
```

Comment:
it might include "." ( example : https://www.boswps.io/#/poll_detail?proposal=hyperion.api )